sdk/communication/azure-communication-email/README.md

```diff
@@ -0,0 +1,176 @@
+```python
+content = EmailContent(
+    subject="This is the subject",
+    plain_text="This is the body",
+    html= "<html><h1>This is the body</h1></html>",
+)
+
+address = EmailAddress(email="customer@domain.com", display_name="Customer Name")
+
+message = EmailMessage(
+            sender="sender@contoso.com",
+            content=content,
+            recipients=EmailRecipients(to=[address])
+        )
+
+response = client.send(message)
+```
+
+### Send an Email Message to Multiple Recipients
+
+To send an email message to multiple recipients, add a object for each recipient type and an object for each recipient.
+
+```python
+content = EmailContent(
+    subject="This is the subject",
+    plain_text="This is the body",
+    html= "<html><h1>This is the body</h1></html>",
+)
+
+recipients = EmailRecipients(
+        to=[
+            EmailAddress(email="customer@domain.com", display_name="Customer Name"),
+            EmailAddress(email="customer2@domain.com", display_name="Customer Name 2"),
+        ],
+        cc=[
+            EmailAddress(email="ccCustomer@domain.com", display_name="CC Customer Name"),
+            EmailAddress(email="ccCustomer2@domain.com", display_name="CC Customer Name 2"),
+        ],
+        bcc=[
+            EmailAddress(email="bccCustomer@domain.com", display_name="BCC Customer Name"),
+            EmailAddress(email="bccCustomer2@domain.com", display_name="BCC Customer Name 2"),
+        ]
+    )
+
+message = EmailMessage(sender="sender@contoso.com", content=content, recipients=recipients)
+response = client.send(message)
+```
+
+### Send Email with Attachments
+
+Azure Communication Services support sending email with attachments.
+
+```python
+file = open("C://readme.txt", "r")
+file_contents = file.read()
+file.close()
+
+content = EmailContent(
+    subject="This is the subject",
+    plain_text="This is the body",
+    html= "<html><h1>This is the body</h1></html>",
+)
+
+address = EmailAddress(email="customer@domain.com", display_name="Customer Name")
+
+attachment = EmailAttachment(
+    name="readme.txt",
+    attachment_type="txt",
+    content_bytes_base64=base64.b64encode(file_contents)
+)
+
+message = EmailMessage(
+            sender="sender@contoso.com",
+            content=content,
+            recipients=EmailRecipients(to=[address]),
+            attachments=[attachment]
+        )
+
+response = client.send(message)
+```
+
+### Get Email Message Status
+
+The result from the `send` call contains a `message_id` which can be used to query the status of the email.
+
+```python
+response = client.send(message)
+status = client.get_sent_status(message_id)
+```
+
+## Troubleshooting
+
+Email operations will throw an exception if the request to the server fails. The Email client will raise exceptions defined in [Azure Core](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/README.md).
+
+```Python
+try:
+    response = email_client.send(message)
+except Exception as ex:
```

Comment:
This snippet should demonstrate importing `HttpResponseError` from azure.core.exceptions and catching that.